### PR TITLE
sundtek: init at 30-06-2015

### PR DESCRIPTION
**Pull request by simonvandel**
Have only tested the 64bit version, but the file structure is the same in the downloaded archive, so I guess it works the same as 64 bit.

### CODE REVIEW

File: nixos/modules/services/misc/sundtek.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  config = mkIf cfg.enable {
+
+    environment.systemPackages = [ pkgs.sundtek ];
+
+    systemd.services.sundtek = {
+      description = "Sundtek driver";
+      wantedBy = [ "multi-user.target" ];
+      wants = [ "graphical.target" ];
```
Comment (lethalman) [inline nixos/modules/services/misc/sundtek.nix]: Why is this needed...

Comment (simonvandel) [inline nixos/modules/services/misc/sundtek.nix]: What in particular do you refer to? The option or the systemd service or something else?

Comment (lethalman) [inline nixos/modules/services/misc/sundtek.nix]: The wants on graphical.target.

Comment (simonvandel) [inline nixos/modules/services/misc/sundtek.nix]: Right, it is taken from upstream, so i don't know if it is really needed. I just tested it on my system, and it works even if removing it.

**Comment by simonvandel** [comment]
Can you take a look again @lethalman ?

**Comment by lethalman** [comment]
The PR cannot be merged.

**Comment by simonvandel** [comment]
Sorry, rebased on master

**Comment by lethalman** [comment]
Looks ok to me, thanks.
